src/asio-individuals.ttl

```diff
@@ -0,0 +1,463 @@
+@prefix : <http://www.asio.es/asioontologies/asio#> .
+@prefix ro: <http://purl.org/wf4ever/ro#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix deo: <http://purl.org/spar/deo/> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix asio: <http://www.asio.es/asioontologies/asio#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@base <http://www.asio.es/asioontologies/asio> .
+
+<http://www.asio.es/asioontologies/asio> rdf:type owl:Ontology ;
+                                          owl:versionIRI <http://www.asio.es/asioontologies/asio/0.0.1> ;
+                                          dct:license <https://creativecommons.org/publicdomain/zero/1.0/> ;
+                                          rdfs:label "ASIO individuals" .
+
+###  http://www.asio.es/asioontologies/asio#BasqueCentreForClimateChange
+asio:BasqueCentreForClimateChange rdf:type owl:NamedIndividual ,
```

Comment:
I don't know if we should use the same prefix ('asio') for the instances, like in this example asio:BasqueCentreForClimate research. Should we create another prefix that points to a different uri (e.g. https://asio.es/asioinstances/asio#) and use that for the instances?

What is your opinion on this @labra @spitxa?